src/main/java/adeuxpas/back/controller/AdController.java

```diff
@@ -0,0 +1,74 @@
+package adeuxpas.back.controller;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.security.core.userdetails.UsernameNotFoundException;
+import jakarta.persistence.EntityNotFoundException;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+import org.springframework.web.bind.annotation.RequestBody;
+import adeuxpas.back.dto.AdPostRequestDTO;
+import adeuxpas.back.service.AdService;
+
+import io.swagger.v3.oas.annotations.responses.*;
+import io.swagger.v3.oas.annotations.Operation;
+
+@RestController
+@RequestMapping("/ad")
```

Comment:
Might be better to use plural names for the api edpoints, like 'ads' instead of 'ad'

Reply:
ok

Reply:
Don't forget to change it in the Front service as well :)